reference.md

```diff
@@ -0,0 +1,1393 @@
+# Snapcraft CLI reference
+
+## lifecycle and misc
+
+### version
+
+    $ snapcraft --version
+    2.12.1
+
+### help
+
+    $ snapcraft help
+    Usage:
+      snapcraft [options] [--enable-geoip --no-parallel-build]
```

Comment:
Having options in all these commands is just noise. I think we can get rid of this when we move out of docopt.